github/ci/prow-deploy/files/jobs/kubevirt/containerdisks/containerdisks-presubmits.yaml

```diff
@@ -123,6 +123,41 @@ presubmits:
             memory: 12Gi
         securityContext:
           privileged: true
+  - always_run: false
+    run_if_changed: "artifacts/debian/.*"
+    annotations:
+      testgrid-create-test-group: "false"
+    cluster: prow-workloads
+    decorate: true
+    decoration_config:
+      grace_period: 5m0s
+      timeout: 1h0m0s
+    labels:
+      preset-podman-in-container-enabled: "true"
+      preset-docker-mirror-proxy: "true"
+    max_concurrency: 11
+    name: pull-containerdisks-pipeline-debian
```

Comment:
Apologies I thought this was already in place when I approved, we really shouldn't land new containerdisks and preferences without pre-submit jobs merged and passing.

Reply:
No issues. Let's get this merged.